config/routes.rb

```diff
@@ -1,21 +1,30 @@
 Rails.application.routes.draw do
+
   devise_for :users
   root "static_pages#home"
   get "/home", to: "static_pages#home", as: "home"
-  get "/learn-more", to: "static_pages#learn_more", as: "learn_more"
-  get "/tools-resources", to: "static_pages#tools_resources", as: "tools_resources"
-  get "/techhire-locations", to: "static_pages#techhire_locations", as: "techhire_locations"
+  get "/learn_more", to: "static_pages#learn_more", as: "learn_more"
+  get "/tools_resources", to: "static_pages#tools_resources", as: "tools_resources"
+  get "/techhire_locations", to: "static_pages#techhire_locations", as: "techhire_locations"
   get '/admin', to: 'sessions#new', as: 'admin/login'
   post '/admin', to: 'sessions#create'
   get '/download', to: "static_pages#download", as: "download"
 
   namespace :admin do
     resources :dashboard, only: [:index]
```

Comment:
So here is my thought. When an admin is adding a resource, I want them to see /admin/pdfs (the index). On this page, they can create/upload a new pdf (the new route is on this page pretty much) and see a list of all pdfs. So the new/create/index resources are taken care of here on the admin/pdfs route.